src/main/java/com/synopsys/integration/alert/workflow/scheduled/PhoneHomeTask.java

```diff
@@ -128,6 +131,20 @@ public String scheduleCronExpression() {
         return CRON_EXPRESSION;
     }
 
+    public Map<String, String> getPhoneHomeVariables() {
+        Map<String, String> environment = System.getenv();
```

Comment:
Should we have an Alert property for this as well and check that first? We're using spring which should be managing our environment.